test/verify/check-system-info

```diff
@@ -895,6 +895,36 @@ password=foobar
         b.wait_in_text(".pf-c-menu__item.pf-m-selected", "Custom crypto policy")
         b.click("#crypto-policy-dialog button.pf-c-button.pf-m-link")
 
+    @skipImage("crypto-policies not avaiable", "fedora-coreos", "debian-stable", "debian-testing", "ubuntu-2004", "ubuntu-stable", "arch")
+    def testFIPSPolicy(self):
+        m = self.machine
+        b = self.browser
+        cmd = "update-crypto-policies"
+
+        # Admin sets FIPS in terminal
+        m.execute(cmd + " --set FIPS")
+        self.login_and_go("/system")
+        b.wait_text("#fips_mode_disabled_text", "FIPS crypto policy selected, but not active")
+        b.click(".system-health-crypto-policies button.pf-c-button.pf-m-link")
+        b.click("#enable-fips-mode-dialog button.pf-c-button.pf-m-primary")
+
+        # fips-mode-setup takes a while to complete.
+        m.wait_reboot(timeout_sec=180 * 3)
+        m.start_cockpit()
+        self.login_and_go("/system")
+        b.wait_text("#crypto-policy-button", "FIPS")
+
+        # Setting via dialog
+        m.execute(cmd + " --set DEFAULT")
```

Comment:
At that point, the kernel still uses FIPS mode, no? So this is again inconsistent -- we should check this both ways. This is also worth a bug report against `update-crypto-policies` perhaps -- trying to switch away from FIPS while the kernel still has it enabled should fail, and point you to `fips-mode-setup`.

Reply:
Yes, but we don't show a message when you have enabled FIPS mode and you don't use fips policy. Maybe that was an misunderstanding from my part?

Reply:
I'm fine with postponing that direction to a follow-up (jira card). It may also make sense to ask the crypto-policies devs what should happen in such a case.